packages/extensions-sdk/package.json

```
@@ -62,7 +62,7 @@
 		"@types/fs-extra": "11.0.4",
 		"@types/inquirer": "9.0.7",
 		"@vitest/coverage-v8": "2.1.2",
-		"typescript": "5.7.2",
+		"typescript": "5.6.3",
```

Was this change on purpose?

```suggestion
		"typescript": "5.7.2",
```